backup/backup.go

```diff
@@ -188,6 +188,181 @@ func (c *BackupCommand) BackupInfo(cliConnection plugin.CliConnection, backupId
 
 }
 
+func (c *BackupCommand) ListBackupsByInstanceGuid(cliConnection plugin.CliConnection, serviceInstanceGuid string) {
+	fmt.Println("Getting the list of  backups in the org", AddColor(helper.GetOrgName(helper.ReadConfigJsonFile()), cyan), "/ space", AddColor(helper.GetSpaceName(helper.ReadConfigJsonFile()), cyan), "/ service instance GUID", AddColor(serviceInstanceGuid, cyan), "...")
+
+	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
+		errors.NoAccessTokenError("Access Token")
+	}
+
+	client := GetHttpClient()
+
+	var userSpaceGuid string = helper.GetSpaceGUID(helper.ReadConfigJsonFile())
+
+	var guid string = serviceInstanceGuid
+	var ServiceName string = guidTranslator.FindInstanceName(cliConnection, guid, nil)
+
+	if ServiceName == "" {
+		errors.IncorrectGuid(guid)
+		return
+	}
+
+	var apiEndpoint string = helper.GetApiEndpoint(helper.ReadConfigJsonFile())
+	var broker string = GetBrokerName()
+	var extUrl string = GetExtUrl()
+
+	apiEndpoint = strings.Replace(apiEndpoint, "api", broker, 1)
+
+	req, err := http.NewRequest("GET", apiEndpoint+extUrl+"/backups"+"?space_guid="+userSpaceGuid, nil)
+	errors.ErrorIsNil(err)
+
+	var resp *http.Response = GetResponse(client, req)
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetBorder(false)
+	table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.SetCenterSeparator(" ")
+	table.SetColumnSeparator(" ")
+	table.SetRowSeparator(" ")
+	table.SetHeaderLine(false)
+	table.SetAutoFormatHeaders(false)
+
+	if resp.Status == "200 OK" {
+		fmt.Println(AddColor("OK", green))
+		//var temp = string(body)
+
+		//var lines = strings.Split(temp, ",")
+		var flag bool
+
+		table.SetHeader([]string{AddColor("backup_guid", white), AddColor("username", white), AddColor("type", white), AddColor("trigger", white), AddColor("started_at", white), AddColor("finished_at", white)})
+
+		var response []interface{}
+		if err := json.Unmarshal(body, &response); err != nil {
+			fmt.Println("Invalid response for the request ", err)
+		}
+
+		var no_of_columns int = 6
+		var field = make([]string, no_of_columns)
+
+		for backup := range response {
+			if strings.Contains(guid, (response[backup].(map[string]interface{}))["instance_guid"].(string)) {
```

Comment:
if the response already contains the backups only belonging to the `instance_guid` specified by the user, then do we need this filtering and comparison? This will consume huge time. This same thing is also happening in `ListBackupsByInstanceName`. Kindly remove in both places.